lib/ts/index.ts

```diff
@@ -91,6 +92,52 @@ export default class AuthHttpRequest {
             userContext: getNormalisedUserContext(input === undefined ? undefined : input.userContext)
         });
     };
+
+    static getInvalidClaimsFromResponse = async function(input: {
+        response: { data: any } | Response;
+        userContext?: any;
+    }): Promise<ClaimValidationError[]> {
+        return AuthHttpRequestFetch.recipeImpl.getInvalidClaimsFromResponse({
+            response: input.response,
+            userContext: getNormalisedUserContext(input.userContext)
+        });
+    };
+
+    static validateClaims = (
+        overrideGlobalClaimValidators?: (
+            globalClaimValidators: SessionClaimValidator[],
+            userContext: any
+        ) => SessionClaimValidator[],
+        userContext?: any
+    ): Promise<ClaimValidationError[]> | ClaimValidationError[] => {
+        const normalisedUserContext = getNormalisedUserContext(userContext);
+        const claimValidatorsAddedByOtherRecipes = AuthHttpRequest.getClaimValidatorsAddedByOtherRecipes();
+        const globalClaimValidators = AuthHttpRequestFetch.recipeImpl.getGlobalClaimValidators({
+            claimValidatorsAddedByOtherRecipes,
+            userContext: normalisedUserContext
+        });
+        const claimValidators =
+            overrideGlobalClaimValidators !== undefined
+                ? overrideGlobalClaimValidators(globalClaimValidators, normalisedUserContext)
+                : globalClaimValidators;
+
+        if (claimValidators.length === 0) {
+            return [];
+        }
```

Comment:
Shouldn't this go inside the recipe implementation file? Similar to how it's done in the node SDK. Or maybe we should make the Node SDK such that it deosn't happen in the recipe implementation.

I think I prefer the approach taken in this SDK cause then it keeps the interface of `validateClaims` cleaner, for when the user is overriding the function.

Reply:
I also prefer this method.
In the node SDK, I moved it into a separate function on the `recipeImplementation` as we've discussed.